packages/vat-data/src/kind-utils.js

```diff
@@ -1,22 +1,25 @@
 import { provide } from '@agoric/store';
-import {
-  defineDurableKind,
-  defineDurableKindMulti,
-  makeKindHandle,
-} from './vat-data-bindings.js';
+import { defineDurableKind, makeKindHandle } from './vat-data-bindings.js';
+
+const { fromEntries, entries } = Object;
+
+export const dropContext =
+  fn =>
+  (_, ...args) =>
+    fn(...args);
+// @ts-expect-error TODO statically recognize harden
+harden(dropContext);
 
 export const provideKindHandle = (baggage, kindName) =>
   provide(baggage, `${kindName}_kindHandle`, () => makeKindHandle(kindName));
 // @ts-expect-error TODO statically recognize harden
 harden(provideKindHandle);
 
-export const provideDurableSingleton = (
-  baggage,
-  kindName,
-  behavior,
-  options = undefined,
-) => {
+export const ProvideFar = (baggage, kindName, methods, options = undefined) => {
   const kindHandle = provideKindHandle(baggage, kindName);
+  const behavior = fromEntries(
+    entries(methods).map(([k, m]) => [k, dropContext(m)]),
+  );
```

Comment:
mapping object values at the existing keys is a common need so in run-protocol package we have a helper 
https://github.com/Agoric/agoric-sdk/blob/68d27f32651a49021d10a50863504e3b92988419/packages/run-protocol/src/collect.js#L6-L8

should we have it in a more shared package? `@agoric/utils`?

Reply:
Yes, I have a similar one, yes with an extensive comment, at 
https://github.com/Agoric/agoric-sdk/blob/86d58b972b123bd77ade5371fd72340da9c6c8a0/packages/zoe/src/objArrayConversion.js#L45-L48

Which of these two do you prefer?

Like `mapValues` it is not reusable only because of where it is. However, `@agoric/utils` would imply that it is in agoric-sdk and so not reusable from the endo repository. Attn @kriskowal is there a natural endo package to migrate such things to? Should there be?

In the meantime, vat-data should clearly not depend on run-protocol or zoe. But I can at least make it a copy of one of these with a note to depend on a common one once a suitable place is found. In the absence of other feedback, I'll copy `objectMap`. Thanks.

Reply:
@turadg I copied `objectMap` but let me know if I should change it to `mapValues`.

Despite the two PTALs above, I'm going to merge this PR into https://github.com/Agoric/agoric-sdk/pull/5283 . But still, please do PTAL and comment either here or (preferred) on https://github.com/Agoric/agoric-sdk/pull/5283 . Thanks.